### PR TITLE
Problem: Incorrect `pip-compile` command in instructions

### PR DESCRIPTION
**Pull request by julianpistorius**
Solution: Removed extra, incorrect file argument

## Checklist before merging Pull Requests
- ~[ ] New test(s) included to reproduce the bug/verify the feature~
- [x] Documentation created/updated at [Example link to documentation](https://example.test/doc#new_section) to give context to the feature
- [ ] Reviewed and approved by at least one other contributor.
- ~[ ] If necessary, include a snippet in CHANGELOG.md~

### CODE REVIEW

File: REQUIREMENTS.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ generated requirements.
 
 Your second command is actually more like:
 ```
-$ pip-compile -o requirements.txt requirements.txt requirements.in
+$ pip-compile -o requirements.txt requirements.in
```
Comment (cdosborn) [inline REQUIREMENTS.md]: The proposed change is incorrect. If this section is unclear, we should change it. In the context of lines 57- 82, I think line 79 makes sense.

Comment (julianpistorius) [inline REQUIREMENTS.md]: Oops. I'll have a closer look.

Comment (julianpistorius) [inline REQUIREMENTS.md]: If I understand this correctly this line is trying to say that:

`pip-compile -o requirements.txt requirements.in`

is equivalent to:

`pip-compile -o requirements.txt requirements.txt requirements.in`

And is _not_ saying that you should run it like this to update the requirements.txt.

Comment (cdosborn) [inline REQUIREMENTS.md]: Yeah

Comment (julianpistorius) [inline REQUIREMENTS.md]: Ah! Thanks. That makes sense. Sorry about the confusion.

**Comment by julianpistorius** [comment]
Closing. #not-a-bug.

**Pull request closed without merging**